### ISSUE TITLE
UserWarning: Unable to import 'webrtcvad'. This package enables noise removal and is recommended.

### ISSUE THREAD

**Issue by mukilansuma** [CorentinJ/Real-Time-Voice-Cloning#745]
(venv) C:\Users\ELECTROBOT\Desktop\voices\Real-Time-Voice-Cloning>python demo_cli.py
C:\Users\ELECTROBOT\Desktop\voices\Real-Time-Voice-Cloning\encoder\audio.py:13: UserWarning: Unable to import 'webrtcvad'. This package enables noise removal and is rec
ommended.
  warn("Unable to import 'webrtcvad'. This package enables noise removal and is recommended.")
Arguments:
    enc_model_fpath:   encoder\saved_models\pretrained.pt
    syn_model_fpath:   synthesizer\saved_models\pretrained\pretrained.pt
    voc_model_fpath:   vocoder\saved_models\pretrained\pretrained.pt
    cpu:               False
    no_sound:          False
    seed:              None
    no_mp3_support:    False

C:\Users\ELECTROBOT\Desktop\voices\Real-Time-Voice-Cloning\venv\lib\site-packages\librosa\core\audio.py:162: UserWarning: PySoundFile failed. Trying audioread instead.
  warnings.warn("PySoundFile failed. Trying audioread instead.")
Librosa will be unable to open mp3 files if additional software is not installed.
Please install ffmpeg or add the '--no_mp3_support' option to proceed without support for mp3 files.

(venv) C:\Users\ELECTROBOT\Desktop\voices\Real-Time-Voice-Cloning>pip list
Package           Version
----------------- ------------
appdirs           1.4.4
audioread         2.1.9
certifi           2020.12.5
cffi              1.14.5
chardet           4.0.0
cycler            0.10.0
decorator         5.0.7
dill              0.3.3
idna              2.10
inflect           5.3.0
joblib            1.0.1
jsonpatch         1.32
jsonpointer       2.1
kiwisolver        1.3.1
librosa           0.8.0
llvmlite          0.36.0
matplotlib        3.4.1
multiprocess      0.70.11.1
numba             0.53.1
numpy             1.19.3
packaging         20.9
Pillow            8.2.0
pip               21.0.1
pooch             1.3.0
pycparser         2.20
pynndescent       0.5.2
pyparsing         2.4.7
PyQt5             5.15.4
PyQt5-Qt5         5.15.2
PyQt5-sip         12.8.1
python-dateutil   2.8.1
pyzmq             22.0.3
requests          2.25.1
resampy           0.2.2
scikit-learn      0.24.1
scipy             1.6.2
setuptools        56.0.0
six               1.15.0
sounddevice       0.4.1
SoundFile         0.10.3.post1
threadpoolctl     2.1.0
torch             1.8.1+cu102
torchaudio        0.8.1
torchfile         0.1.0
torchvision       0.9.1+cu102
tornado           6.1
tqdm              4.60.0
typing-extensions 3.7.4.3
umap-learn        0.5.1
Unidecode         1.2.0
urllib3           1.26.4
visdom            0.1.8.9
websocket-client  0.58.0

**Comment by blue-fish** [comment]
The relevant part of the error message:
```
Librosa will be unable to open mp3 files if additional software is not installed.
Please install ffmpeg or add the '--no_mp3_support' option to proceed without support for mp3 files.
```

**Comment by blue-fish** [comment]
See steps 9 and 10 of our [Windows setup instructions](https://github.com/CorentinJ/Real-Time-Voice-Cloning/pull/642#issuecomment-768900542) if you have trouble installing FFmpeg.

**Comment by Vasanthan-aka-Maari** [comment]
I'm having the same error and unable to solve it

**Comment by Vasanthan-aka-Maari** [comment]
> 
Still I'm facing this issue... even after install ffmpeg
![image](https://user-images.githubusercontent.com/67197973/225083764-1473a680-7c52-491e-8cce-94c96e5854b2.png)
